assets/email_template.html

```diff
@@ -4,7 +4,7 @@
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1">
 
-  <meta name="description" content="nf-core/variantbenchmarking: Nextflow variant benchmarking pipeline">
+  <meta name="description" content="nf-core/variantbenchmarking: Master benchmarking pipeline for Structural Variant callers ">
```

Comment:
the term "master" in the sense of ruler, has some bad connotation. Maybe a different term would fit better?

Reply:
Good idea! We'll be sure to fix this before the release